src/shared/credentials/userCredentialsUtils.ts

```diff
@@ -98,25 +75,13 @@ export class UserCredentialsUtils {
         const credentialsFileContents = handlebarTemplate(credentialsContext)
 
         // Make a final check
-        if (await SystemUtilities.fileExists(this.getCredentialsFilename())) {
+        if (await SystemUtilities.fileExists(getCredentialsFilename())) {
             throw new Error('Credentials file exists. Not overwriting it.')
         }
 
-        await writeFile(this.getCredentialsFilename(), credentialsFileContents, {
+        await writeFile(getCredentialsFilename(), credentialsFileContents, {
             encoding: 'utf8',
             mode: 0o100600 // basic file (type 100) with 600 permissions
```

Comment:
why do we have an octal number in our code base

Reply:
/shrug